src/Client.php

```diff
@@ -25,12 +25,12 @@ public function __construct(
         }
         $stack = $config['handler'];
 
-        // Download file without decoding
-        $config['decode_content'] = false;
```

Comment:
Tymto sa povodne fixol prvy problem: `Unrecognized content encoding type. libcurl understands deflate, gzip content encodings.` ale rozbilo sa to inym klientom, ktory sa na dekodovanie spoliehali.
Teda som to revertol.